packages/markdown-pdf/src/PdfTransformer.test.js

```diff
@@ -151,6 +163,42 @@ describe('pdf import 2', () => {
     });
 });
 
+describe('pdf generation with decorators', () => {
+    it.only('converts signature-block.md', async () => {
+        const templateTransformer = new TemplateMarkTransformer();
+        const templateMarkContent = fs.readFileSync( path.join(__dirname, '/../test/data/signature', 'grammar.tem.md'), 'utf-8' );
+
+        const modelManager = templatemarkutil.templateMarkManager.modelManager;
```

Comment:
you should be able to use `const modelManager = await ModelLoader.loadModelManager(null, models, options);` to replace a lot to this, and remove the `AP_SYSTEM_MODEL`.